packages/sdk/src/HopBridge.ts

```diff
@@ -2240,6 +2255,122 @@ class HopBridge extends Base {
     this.priceFeedApiKeys = apiKeys
     this.priceFeed.setApiKeys(this.priceFeedApiKeys)
   }
+
+  private async getRelayerFee (destinationChain: TChain): Promise<BigNumber> {
+    // TODO: Introduce this post-nitro
+    // if (destinationChain === Chain.Arbitrum) {
+    //   return this.getArbitrumRelayGasCost(destinationChain)
+    // }
+
+    return BigNumber.from(0)
+  }
+
+  private async getArbitrumRelayGasCost (destinationChain: TChain): Promise<BigNumber> {
```

Comment:
suggestion: same comment as in hop-node as to move this logic to a util so it can be independently unit tested and to make it easier to copy/paste from hop-node<>sdk since the logic looks to be the same

Reply:
Resovled in 6f6cc07f0a419b7c2d18accbd3969aa1e951e472